src/components/Counter/Counter.tsx

```diff
@@ -59,6 +70,21 @@ export class Counter extends React.Component<CounterProps> {
     onChange((value + step).toString());
   };
 
+  _getPlus = () => {
+    const { size } = this.props;
+    return size === CounterSize.xSmall ? <PlusS /> : <Plus />;
+  }
+
+  _getMinus = () => {
+    const { size } = this.props;
+    return size === CounterSize.xSmall ? <MinusS /> : <Minus />;
+  }
```

Comment:
Is it necessary to use different assets for the different sizes?
Or maybe we can just change it via css (which is better)?

Reply:
done